ci/tox/build-context/entrypoint.sh

```diff
@@ -8,7 +8,12 @@
 set -e -u -x
 
 $PYTHON -m pip install --user tox
-TOXENV="pep8,py${PYTHON_VERSION/./}" tox
+
+if [[ $CHECK_COVERAGE = true ]]; then
+  TOXENV="pep8,py${PYTHON_VERSION/./},coverage" tox
```

Comment:
Also! since you've mentioned `-e` option to `tox`... would using this option instead of `TOXENV` improve readability here?

Reply:
I think that it's a good idea.